Casks/tomatobar.rb

```diff
@@ -0,0 +1,11 @@
+cask "tomatobar" do
+  version "3.3.0"
+  sha256 "ff4d465f0d70cb2e234918a650d3844352d0179e9f3742666ae98d909988dc23"
+
+  url "https://github.com/ivoronin/TomatoBar/releases/download/v#{version}/TomatoBar-v#{version}.zip"
+  name "tomato-bar"
+  desc "🍅 World's neatest Pomodoro timer for menu bar"
```

Comment:
From the Cask cookbook - [Do not use empty marketing jargon.](https://docs.brew.sh/Cask-Cookbook#stanza-desc)
```suggestion
  desc "Menu bar pomodoro timer"
```